### PR TITLE
{astro}[foss/2021b] emcee v3.1.4 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by VRehnberg**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by VRehnberg** [comment]
There seem to no longer be a `emcee.test` attribute. So I figured `pytest` would work instead. But, I'm unsure about this so that might be something to look extra closely at.

**Comment by VRehnberg** [comment]
Test report by @VRehnberg
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.7, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/VRehnberg/2f23fc0f4f21efdefdb64900e4ba3d20 for a full test report.

File: easybuild/easyconfigs/e/emcee/emcee-3.1.4-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'emcee'
+version = '3.1.4'
+
+homepage = 'https://dfm.io/emcee'
```
Comment (branfosj) [inline easybuild/easyconfigs/e/emcee/emcee-3.1.4-foss-2021b.eb:6]:
```suggestion
homepage = 'https://emcee.readthedocs.io/'
```

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/cca8f0b788bb07b8baf61a5c286be75c for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @VRehnberg!

**Pull request merged**